src/Api/Entity.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Friendica\Api;
+
+/**
+ * The API entity classes are meant as data transfer objects. As such, their member should be protected.
+ * Then the JsonSerializable interface ensures the protected members will be included in a JSON encode situation.
+ *
+ * Constructors are supposed to take as arguments the Friendica dependencies/model/collection/data it needs to
+ * populate the class members.
+ */
+abstract class Entity implements \JsonSerializable
```

Comment:
But these entities aren't meant for further processing since they are used to represent data for a target API. The constructor is all the processing they need.